dockstore-webservice/src/main/java/io/dockstore/webservice/resources/CollectionResource.java

```diff
@@ -401,4 +409,16 @@ private OrganisationUser getUserOrgRole(Organisation organisation, Long userId)
                 .equals(organisationUser.getUser().getId(), userId)).findFirst();
         return matchingUser.orElse(null);
     }
+
+    /**
+     * Checks if the include string (csv) includes some field
+     * @param include CSV string
+     * @param field Field to query for
+     * @return True if include has the given field, false otherwise
+     */
+    private boolean checkIncludes(String include, String field) {
+        String includeString = (include == null ? "" : include);
+        ArrayList<String> includeSplit = new ArrayList(Arrays.asList(includeString.split(",")));
```

Comment:
1. You should just be able to use the value returned by `Arrays.asList` directly, without creating a new ArrayList on top of that.
2. The CSV string will not handle more complicated CSV formatting. I think that's OK, but I don't entirely understand your input, so just making sure. An example is a CSV value with a comma in it, in which case the value has to be double quoted. I know this because I had an interview question once where I had to parse a CSV file. :) Like I said, probably not an issue here, but mentioning it just in case.

Reply:
1. :+1: 
2. So the general idea is the includes is a list of strings separated by commas. These strings should only be made from letters of the alphabet. I think this is a sign that I should better document, so I will add more documentation to this.